commands/next_lint_error.py

```diff
@@ -57,16 +57,25 @@ def _harvest_next(self) -> int:
             self.window.active_view().sel()[0].begin()
         )
         lines = set([])
+        next_change = None
         vid = self.window.active_view().id()
         for error_type in ['ERRORS', 'WARNINGS', 'VIOLATIONS']:
             for line, _ in ANACONDA[error_type].get(vid, {}).items():
                 lines.add(int(line))
 
-        lines = set(sorted(lines))
+        lines = list(set(sorted(lines)))
         if not len(lines):
             return None
 
-        if cur_line and list(lines)[-1] > cur_line:
-            lines_list = [l for l in lines if l > cur_line]
+        if not cur_line:
```

Comment:
We don't really need all this code block at all, the only thing that we need is define `lines_list` in line 65 just before the conditional, like:
```python
if not len(lines):
    return None

lines_list = []
if cur_line and list(lines)[-1] > cur_line:
```